hcloud/hcloud.py

```diff
@@ -32,6 +32,8 @@ class HcloudClient(object):
     def __init__(self, token):
         self.token = token
         self.api_endpoint = "https://api.hetzner.cloud/v1"
+        self.application_name = ""
```

Comment:
Using `None` to denote the absence of an application name would be more pythonic.